database/seeds/BlocksTableSeeder.php

```diff
@@ -89,6 +89,8 @@ public function run()
                 <li><a title="Skontaktuj się poprzez email" href="https://www.facebook.com/4programmers.net"><i class="fa fa-facebook fa-fw"></i> Odwiedź nas na Facebooku</a></li>
             </ol>
         </div>
+        
+        <a id="back-to-top" href="#top" title="Powrót na górę">Powrót na górę</a>
```

Comment:
byc moze zamiast ikony w formie graficznej, mozna by bylo uzyc tutaj font awesome? uzywamy font awesome w kazdem miejscu serwisu, zamiast grafiki. proponuje to: http://fontawesome.io/icon/angle-up/

czyli tutaj kod html wygladalby tak:

```html
<a id="back-to-top" href="#top" title="Powrót na górę"><i class="fa fa-angle-up"></i></a>
```

spowodowaloby to wyswietlenie w tym miejscu strzalki, tyle, ze koniecznosci uzycia grafiki :)